src/Components/ArticleCard/ArticleCard.jsx

```diff
@@ -0,0 +1,47 @@
+import { React, useState } from "react";
+import { Link } from "react-router-dom";
+import { Icon } from "@iconify-icon/react";
+import styles from "./ArticleCard.module.scss";
+import TagItem from "../TagItem/TagItem";
```

Comment:
Export TagItem in index.jsx of Components directory and import as

`import {TagItems} from "../../Components"`